src/lib/formatter/SarifFormatter.ts

```diff
@@ -247,7 +249,8 @@ class RetireJsSarifFormatter extends SarifFormatter {
 				tool: {
 					driver: {
 						name: 'Retire.js',
-						version: retire.version,
+						// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
`retire` has no associated `@types` package, so we just have to suppress the violation here. No easy way around that.